client/extensions/woocommerce/state/sites/request.js

```diff
@@ -31,6 +31,7 @@ const _request = ( method, path, siteId, body ) => {
 		{
 			path: `/wc/v3/${ path }&_method=${ method }`,
 			body: body && JSON.stringify( body ),
+			json: true,
```

Comment:
Huh, really? It worked for me without `json: true`... that's weird... but it doesn't hurt.